plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/standard/StandardDeployCommandHandler.java

```diff
@@ -147,55 +141,15 @@ private StandardDeployJobConfig getDeployJobConfig(IProject project,
     return config;
   }
 
-  private DefaultDeployConfiguration getDeployConfiguration(IProject project, ExecutionEvent event)
-      throws ExecutionException, DeployCancelledException {
+  private DefaultDeployConfiguration getDeployConfiguration(IProject project,
+                                                            ExecutionEvent event) throws ExecutionException {
     StandardDeployPreferences deployPreferences = new StandardDeployPreferences(project);
-    if (deployPreferences.isPromptForProjectId()
-        || deployPreferences.getProjectId() == null
-        || deployPreferences.getProjectId().isEmpty()) {
-      String projectId = promptForProjectId(HandlerUtil.getActiveShellChecked(event), deployPreferences.getProjectId());
-      deployPreferences.setProjectId(projectId);
-    }
-
     if (deployPreferences.getProjectId() == null || deployPreferences.getProjectId().isEmpty()) {
       throw new ExecutionException(Messages.getString("error.projectId.missing"));
     }
     return new StandardDeployPreferencesConverter(deployPreferences).toDeployConfiguration();
   }
 
-  private String promptForProjectId(Shell shell, String initialValue) throws DeployCancelledException {
-    InputDialog dialog = new InputDialog(shell,
-                    Messages.getString("dialog.prompt.projectId.title"),
-                    Messages.getString("dialog.prompt.projectId.message"),
-                    initialValue,
-                    new ProjectIdValidator() {
-                      @Override
-                      // The input dialog does not provide decoration for the error message,
-                      // make errors more recognizable with modified validation messages
-                      // FIXME https://github.com/GoogleCloudPlatform/cloud-tools-for-eclipse/issues/635
-                      public String isValid(String newText) {
-                        String error = super.isValid(newText);
-                        if (error != null) {
-                          if (error.isEmpty()) {
-                            return Messages.getString("error.projectId.invalid");
-                          } else {
-                            return Messages.getString("error.projectId.prefix", error);
```

Comment:
I see no usage of `error.projectId.prefix elsewhere`. Removing from `message.properties`?